packages/amplication-server/src/core/resource/dto/ResourceWhereInput.ts

```diff
@@ -35,4 +35,7 @@ export class ResourceWhereInput {
     description: undefined
   })
   project?: WhereUniqueInput | null;
+
+  @Field(() => String, { nullable: true })
+  projectId?: string | null;
```

Comment:
why did you add `projectId` while we already have the property `project`

Reply:
To match more closely to the Prisma orm